lib/inferno/apps/cli/services.rb

```diff
@@ -3,7 +3,17 @@ module CLI
     class Services < Thor
       no_commands do
         def base_command
-          'docker compose -f docker-compose.background.yml'
+          if bundle_exec? && in_test_kit?
+            compose_path = File.join(test_kit_root, 'docker-compose.background.yml')
+          else
+            if in_test_kit?
+              puts 'Warning: Using global inferno services because user did not run command with `bundle exec`'
```

Comment:
Why does running with `bundle exec` determine whether the user is in a test kit directory?

Reply:
You need `Bundler.load_spec` to parse the gemspec for `in_test_kit?`, but `Bundler` only gets included if you invoke the Ruby code with `bundle exec`.

From a user standpoint I also think its nice to have `bundle exec inferno...` represent commands on the local test kit and `inferno ...` run commands without a test kit.